Easydict/Feature/Configuration/NewConfiguration.swift

```diff
@@ -0,0 +1,565 @@
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.autoCopySelectedText) { [unowned self] _ in
+            didSetAutoCopySelectedText()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.autoCopyOCRText) { [unowned self] _ in
+            didSetAutoCopyOCRText()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.autoCopyFirstTranslatedText) { [unowned self] _ in
+            didSetAutoCopyFirstTranslatedText()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.languageDetectOptimize) { [unowned self] _ in
+            didSetLanguageDetectOptimize()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.defaultTTSServiceType) { [unowned self] _ in
+            didSetDefaultTTSServiceType()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.showGoogleQuickLink) { [unowned self] _ in
+            didSetShowGoogleQuickLink()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.showEudicQuickLink) { [unowned self] _ in
+            didSetShowEudicQuickLink()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.showAppleDictionaryQuickLink) { [unowned self] _ in
+            didSetShowAppleDictionaryQuickLink()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.hideMenuBarIcon) { [unowned self] _ in
+            didSetHideMenuBarIcon()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.enableBetaNewApp) { [unowned self] _ in
+            didSetEnableBetaNewApp()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.fixedWindowPosition) { [unowned self] _ in
+            didSetFixedWindowPosition()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.mouseSelectTranslateWindowType) { [unowned self] _ in
+            didSetMouseSelectTranslateWindowType()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.shortcutSelectTranslateWindowType) { [unowned self] _ in
+            didSetShortcutSelectTranslateWindowType()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.adjustPopButtonOrigin) { [unowned self] _ in
+            didSetAdjustPopButtomOrigin()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.allowCrashLog) { [unowned self] _ in
+            didSetAllowCrashLog()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.allowAnalytics) { [unowned self] _ in
+            didSetAllowAnalytics()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.clearInput) { [unowned self] _ in
+            didSetClearInput()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.fontSizeOptionIndex) { [unowned self] _ in
+            didSetFontSizeIndex()
+        }.tieToLifetime(of: self)
+
+        Defaults.observe(.appearanceType) { [unowned self] change in
+            let newValue = change.newValue
+            let oldValue = change.oldValue
+            print("update appearanceType: \(oldValue.title) -> \(newValue.title)")
```

Comment:
Actually, what I mean is the `Defaults.observe` function will be called twice, and print "update appearanceType:" twice, not just `.appearanceType`, all of the other keys are like this.

It's strange, maybe `Defaults` uses it in a wrong way somewhere, but I don't know it.

Reply:
When I switch from dark mode to follow system

```
update appearanceType: 深色模式 -> 跟随系统
深色模式
update appearanceType: 深色模式 -> 跟随系统
深色模式
```

https://github.com/tisfeng/Easydict/assets/25194972/f9dd25b6-9194-41b8-9fbe-2b67368f4234

Reply:
@CanglongCl Can you take a look at this issue? We need to merge this PR as early as possible.